src/locale/it.ts

```diff
@@ -0,0 +1,118 @@
+export default {
+  partySelection: {
```

Comment:
Verify if each key having the pattern "{{variable}}" has been rollbacked, and remove it from this file